routes/exchange.go

```diff
@@ -1617,41 +1618,63 @@ func (fes *APIServer) GetPostsForFollowFeedForPublicKey(bav *lib.UtxoView, start
 		return nil, errors.Wrapf(err, "GetPostsForFollowFeedForPublicKey: Problem filtering out restricted public keys: ")
 	}
 
-	minTimestampNanos := uint64(time.Now().UTC().AddDate(0, 0, -2).UnixNano()) // two days ago
-	// For each of these pub keys, get their posts, and load them into the view too
-	for _, followedPubKey := range filteredPubKeysMap {
+	var postEntriesForFollowFeed []*lib.PostEntry
+	n := 1
+	for n < 6 {
```

Comment:
`for n := 1 ; n < 6; n++; {`
Also can you find a better variable name for readability